pkgs/tools/graphics/zbar/default.nix

```diff
@@ -1,6 +1,7 @@
 { stdenv
 , lib
 , fetchFromGitHub
+, fetchpatch
```

Comment:
Can be removed / not added.

Reply:
Will probably need it when upstream adds more patches but takes some time to do the release (I am moderately optimistic, I guess)…

Reply:
Damnit I'm always doing this.